test/sdn_migration/sdn_migration_test.go

```diff
@@ -156,19 +157,18 @@ var _ = Describe("SDN migration", Ordered, func() {
 	})
 
 	It("rosa cluster is healthy post 4.15.8 upgrade", postUpgradeCheck, func(ctx context.Context) {
-		err := osdClusterReadyHealthCheck(ctx, testRosaCluster.client, "post-upgrade", testRosaCluster.reportDir)
+		err := cluterOperatorHealthCheck(ctx, testRosaCluster.client, logger)
```

Comment:
I understand the reasoning behind validating all cluster operators are healthy, but can you explain why we no longer want to call `osdClusterReadyHealthCheck` as part of this change?

Reply:
An "all COs ≟ healthy" check effectively includes (or is at least more comprehensive than) the checks performed by osd-cluster-ready. @reedcort please correct me if I'm wrong

Reply:
Yeah, that's correct. The reason I'm not using the 'osdClusterReadyHealthCheck' is that after each upgrade, the nodes will roll and temporarily become unschedulable, which causes certain COs to go from 'Ready' to 'Progressing.' When there are many nodes, the health check job will time out.